br/pkg/lightning/metric/metric.go

```diff
@@ -134,6 +135,13 @@ func NewMetrics(factory promutil.Factory) *Metrics {
 		//  - finished
 		//  - failed
 
+		RowsCounter: factory.NewCounterVec(
+			prometheus.CounterOpts{
+				Namespace: "lightning",
+				Name:      "rows",
+				Help:      "count of total rows",
+			}, []string{"state"}),
```

Comment:
It seems `state` is always `StateTotalRestore`. It's better to change `state` to `table`.

Reply:
It will also use `StateRestored`. It's mainly used to calculate the restored rows progress.

Reply:
In https://github.com/pingcap/tidb/pull/42136, I need table rows to calculate add index progress. Do you think we should use the same counter?

https://github.com/pingcap/tidb/pull/42136/files#diff-5db25be2a353cef3ef986c7ef1abdea086d68cc9ae392448d0f7630cf929d55eR137

Reply:
Yes. I think we should use the same counter. How about keeping both labels?

Reply:
LGTM.

Reply:
I take a second check. It seems that it's hard to calculate the summary of all the counters unless we traverse all the label names of (state, tableName) pairs. Maybe using two different counters is more simple.